docker-compose.yml

```diff
@@ -39,6 +39,24 @@ services:
         VERSION: 7.4
         BASEOS: bullseye
 
+  php80-fpm-buster-base:
+    image: my127/akeneo:8.0-fpm-buster${TAG_SUFFIX:-}
+    build:
+      context: ./
+      dockerfile: base.Dockerfile
+      args:
+        VERSION: "8.0"
+        BASEOS: buster
```

Comment:
lets not bother with buster till we have a real reason to use it

Reply:
each image build adds additional time, so we don't want to increase that for unnecessary images

Reply:
Understood